### DISCUSSION TITLE
TileSource: check if the map was loaded correctly

### DISCUSSION THREAD

**Discussion by moiatto** [mapsforge/vtm#1114] [Q&A] [answered]
I would like to check whether the map was imported correctly via
`TileSource.OpenResult openResult = tileSource.open();`

but I don't understand how and in what position to insert it, because if I add it after `tileSource.setMapFileInputStream` no map loaded:
```
.....
FileInputStream fis = (FileInputStream) context.getContentResolver().openInputStream(mapFile.getUri());
MapFileTileSource tileSource = new MapFileTileSource();
tileSource.setMapFileInputStream(fis);

TileSource.OpenResult openResult = tileSource.open();
.....
```

**Comment by devemux86** [comment, marked as answer]
To check if a map file is valid, can use something like this: open / close it.

To load map files see the GettingStarted or MapsforgeActivity examples.
(with a try / catch)

**Reply by moiatto** [reply]
Ok. Thanks
